jina/parsers/client.py

```diff
@@ -36,7 +36,7 @@ def mixin_client_cli_parser(parser):
     gp.add_argument(
         '--top-k',
         type=int,
-        help='The number of results will be returned. Sorted by their scores descendingly.',
+        help='The number of results will be returned. Sorted by their scores in descending order.',
```

Comment:
We should remove this part, it is not guaranteed to be returned sorted in descending order

Reply:
wow okay, I didn't know that. Interesting.
Is it because the rankers can change the sorting?

Reply:
well, it is because it depends if u sort or not them, or if they come right after a Vector Search (where score is distance)